srcs/juloo.keyboard2/Config.java

```diff
@@ -59,6 +61,7 @@ public final class Config
   public boolean autocapitalisation;
   public boolean switch_input_immediate;
   public boolean pin_entry_enabled;
+  public boolean borderConfig;
```

Comment:
This option might seem like a hack but is a foundation for a future "Custom theme" option. Ideally it would work by creating a `Theme` object and not by adding conditions in the rendering code. Is anyone interested in working on that?